src/manifolds/Grassmann.jl

```diff
@@ -114,16 +114,16 @@ function check_tangent_vector(
         kwargs...,
     )
     mpv === nothing || return mpv
-    if !isapprox(p' * X + X' * p, zeros(k, k); kwargs...)
+    if !isapprox(p' * X, -conj(X' * p); kwargs...)
         return DomainError(
-            norm(p' * X + X' * p),
+            norm(p' * X + conj(X' * p)),
             "The matrix $(X) does not lie in the tangent space of $(p) on $(M), since p'X + X'p is not the zero matrix.",
         )
     end
     return nothing
 end
 
-decorated_manifold(M::Grassmann{N,K,𝔽}) where {N,K,𝔽} = Euclidean(N, K; field = 𝔽)
+decorated_manifold(::Grassmann{N,K,𝔽}) where {N,K,𝔽} = Euclidean(N, K; field = 𝔽)
```

Comment:
A number of lines add the `M` variable to functions where it is not used. I actually prefer to remove variable names where unused unless it makes the code much less clear so it's more obvious when an arg is used only for dispatch, but I don't think we've agreed on a standard. Thoughts?

Reply:
Yes, I am currently removing occurrences of unused variables, whenever I notice them. I also think that it's nicer without stating unused variables.
I just was not aware of that, say, half a year ago; but now they are even nicely grayed out in VS Code, so they are easy to spot.

Reply:
Oh I see now, I read the diff backwards. Yes we are on the same page.